### ISSUE TITLE
Cannot navigate completions when signature help is active

### ISSUE THREAD

**Issue by Pajn** [Microsoft/vscode#21831]
- VSCode Version: 1.10.1
- OS Version: Ubuntu 16.04

Steps to Reproduce:

1. Use a language that provides signature help as well as code completion (for example typescript)
2. Type a function name and the opening parenthesis 
3. Notice the signature help open

4a. Type a letter to start completion
5a. Try to navigate the completion list using the up or down arrow
6a. Notice how the cursor moves instead of the selected completion item

4b. Press ctrl+space to start completion
5b. Try to navigate the completion list using the up or down arrow
6b. Notice how the cursor moves instead of the selected completion item

4c. Press escape to close the signature help
5c. Type a letter to start completion
6c. Try to navigate the completion list using the up or down arrow
7c. The selected item in the completion list now changes, the cursor stays still

I would expect to always be able to navigate the completion list using the arrow keys, or
to always be able to move the cursor (and then be able to navigate the completion list
with something like ctrl+arrow keys). The current non-consistent behavior makes it common
to erroneously move the cursor (and close the completions) when you meant to select an
item in the completion list.

I made a little video to try to demonstrate the problem but it might be hard to follow
without knowing the keypresses so please try the three branches in the above steps 
if you don't see what I mean.
https://www.youtube.com/watch?v=386OJAtxX0o

**Comment by joaomoreno** [comment]
Fixed.
